agents/lakeshore372/LS372_agent.py

```diff
@@ -753,10 +784,12 @@ def make_parser(parser=None):
                               second if it is set longer than a channel's dwell\
                               time. This ensures at least one second of data\
                               collection at the end of a scan.")
-    pgroup.add_argument('--auto-acquire', type=bool, default=True,
+    pgroup.add_argument('--auto-acquire', action='store_true',
                         help='Automatically start data acquisition on startup')
     pgroup.add_argument('--sample-heater', type=bool, default=False,
                         help='Record sample heater output during acquisition.')
+    pgroup.add_argument('--enable-control-chan', action='store_true',
+                        help='Enable reading of the control input each acq cycle')
```

Comment:
This argument should also be added to the ocs-site-config example block in the docs.

Reply:
Are you sure? I don't think labs will typically using that argument, so I'm not sure if it should go into the example block.

Reply:
It should still show up in the argparse autodoc though.

Reply:
> Are you sure? I don't think labs will typically using that argument, so I'm not sure if it should go into the example block.

I've been trying to put every available option in the example block with whatever the default value is. Options like this are, of course, not mandatory, but if someone wants to use it I want there to be a good example.